### DISCUSSION TITLE
List all events with all its attendees

### DISCUSSION THREAD

**Discussion by orcofthedarkness** [liedekef/events-made-easy#114] [Q&A] [answered]
Hi there,

i am running a page (https://www.motosupportcrew.at) organizing a group of motorcyclists supporting events (like Ironman, Triathlons and UCI bike marathons, etc).
We are organized using EME and it works like a charm. :)

I was asked by one of our teamleads, to have a single page where I can list all events WITH its attendees...
This could be used to have a quick overview of the team who is where an when. Unfortunatley I couldn't find such an option or possibility. 
What I'd like to have is kind of this:
 [eme_events show_attendees=1] or so
 
 And this could be displayed as a table? Like (forget the " -------- ")
| Event1 | attendee 1
| --------| attendee 2
| Event2 | attendee 1
|-------- | attendee 4
| Event3 | attendee 3

Did I miss that? Is there a functionality like that?

thank you in advance,
Ron

**Comment by liedekef** [comment, marked as answer]
Use [eme_events] with a header/footer/per-event template that gives you a table of event1/... and then the per event-template also to use either #_ATTENDEES (and change the layout in EME settings) or [eme_attendees] (with the correct event id: id=#_EVENTID), and a template to again show a table with the attendees.

**Comment by orcofthedarkness** [comment]
That is great. Works as needed. Just need to play with the layout now. thx allot

**Reply by GuiguiBoulot** [reply]
Hi @orcofthedarkness ,

I also want to display a list of attendees for my events.
Can you explain what you set in templates (header / footer) and how you call it from your page ?

thank you in advance,
Guigui
